File: front-end/src/app/service-models/app/v1/client/documents-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class DocumentsSM extends CoinManagementServiceModelBase<number> {
    override id!: number;
    name!: string;
    description!: string;
    letterData!: string;
    extension!: string;
    clientCompanyDetailId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/DocumentsSM.cs
using Siffrum.Web.Payroll.ServiceModels.Base;

namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class DocumentsSM : SiffrumPayrollServiceModelBase<int>
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string LetterData { get; set; }
        public string Extension { get; set; }
        public int ClientCompanyDetailId { get; set; }
    }
}
